NineChronicles.Standalone/Controllers/GraphQLController.cs

```diff
@@ -44,6 +45,10 @@ public IActionResult RunStandalone()
             try
             {
                 IHostBuilder nineChroniclesNodeHostBuilder = Host.CreateDefaultBuilder();
+                if (StandaloneContext.NineChroniclesNodeService is null)
+                {
+                  return new StatusCodeResult(StatusCodes.Status409Conflict);
```

Comment:
Shouldn't this be 403 according to the [description](https://github.com/planetarium/NineChronicles.Standalone/pull/17#issue-467876642)?